firmware/screen/src/ui/toast.cpp

```
@@ -0,0 +1,64 @@
+namespace {
+
+constexpr int TOAST_H        = 36;
+constexpr int TOAST_PAD_X    = 16;
+constexpr int TOAST_BOTTOM_OFFSET = 8;
+
+// Per-screen toast — we deliberately do NOT hold a static cross-screen
+// pointer because `ui_nav_pop()` deletes the previous screen and would
+// invalidate any cached widget. Each call creates a fresh toast on the
+// currently active screen and arms a one-shot timer to delete it.
+
+void toast_delete_cb(lv_timer_t* t) {
```

Clang (14.0.6)

[warning] parameter name 't' is too short, expected at least 3 characters

(readability-identifier-length)